app/src/main/kotlin/org/eurofurence/connavigator/ui/LoginActivity.kt

```diff
@@ -132,17 +132,17 @@ class LoginUi : AnkoComponent<LoginActivity> {
 
             layoutMain = verticalLayout {
 
-                // If not logged in
-                verticalLayout {
-                    visibility = if (AuthPreferences.isLoggedIn()) View.GONE else View.VISIBLE
-
                 toolbar {
                     title = "Login"
                     lparams(matchParent, wrapContent)
                     backgroundResource = R.color.primary
                     setTitleTextAppearance(ctx, android.R.style.TextAppearance_Medium_Inverse)
                 }
 
+                // If not logged in
+                verticalLayout {
```

Comment:
Nesting; toolbar was a child of the verticalLayout that only showed for 'login' - now it's 1 level above and shows for the login/logout screens. GitHub sucks diffing nesting changes.